devlab/jenkins/run_functional_tests.sh

```diff
@@ -2,36 +2,31 @@
 set -e
 set -x
 
-WORKSPACE="${WORKSPACE:-$( cd $( dirname "$0" ) && cd ../../../ && pwd)}"
-
-CF_DIR=$WORKSPACE/cloudferry
-JOB_NAME="${JOB_NAME:-cloudferry-functional-tests}"
-
-trap 'clean_exit $LINENO $BASH_COMMAND; exit' SIGHUP SIGINT SIGQUIT SIGTERM EXIT
-clean_exit()
-{
-    pushd ${CF_DIR}/devlab
-    vboxmanage list vms
-    vagrant destroy --force
-    popd
-}
```

Comment:
Please, don't do this. Just keep it in the script and use copy in the job if needed.